src/Neo/SmartContract/Native/StdLib.cs

```diff
@@ -198,6 +199,18 @@ public static byte[] Base58CheckDecode([MaxLength(MaxInputLength)] string s)
             return Base58.Base58CheckDecode(s);
         }
 
+        [ContractMethod(Hardfork.HF_Faun, CpuFee = 1 << 16, RequiredCallFlags = CallFlags.None)]
```

Comment:
```suggestion
        [ContractMethod(Hardfork.HF_Faun, CpuFee = 1 << 5, RequiredCallFlags = CallFlags.None)]
```
Due to my test, the cost time is very closed to `base64encode`, so `1<<5` is correct.